### DISCUSSION TITLE
Приветствую. Прошу помочь и уточнить куда вписывать API-ключ? Уже час потратил, не могу найти это поле.

### DISCUSSION THREAD

**Discussion by IATkachenko** [IATkachenko/HA-YandexWeather#35] [Q&A] [answered]
Приветствую. Прошу помочь и уточнить куда вписывать API-ключ? Уже час потратил, не могу найти это поле.

_Originally posted by @maks23rus in https://github.com/IATkachenko/HA-YandexWeather/issues/33#issuecomment-1166266644_

**Comment by IATkachenko** [comment, marked as answer]
@maks23rus, одно issue -- одна проблема.  Ваш вопрос утащил в отдельное issue.
И, поскольку это -- не проблема интеграции, перекинул в discussions.

Вы пропустили шаг 5, раздела configuration из [README](https://github.com/IATkachenko/HA-YandexWeather/blob/main/README.md)
```
Go to Home Assistant settings

    Integrations
    Add
    Start typing "Yandex weather" (clean browser cache if nothing found)
    Add integration
    Put API key into API key field

```

**Reply by Nikolay-Ch** [reply]
Установил компонент через HACS но он не появляется в интеграциях... Версия HA: 2023.11.0 (docker)
Кэш сброшен

**Reply by IATkachenko** [reply]
@Nikolay-Ch 
![изображение](https://github.com/IATkachenko/HA-YandexWeather/assets/17473830/648814a0-bfdb-4e87-a106-2b32bacceec7)
![изображение](https://github.com/IATkachenko/HA-YandexWeather/assets/17473830/f8237339-a9b4-41d9-85a8-178d0720b3bb)
![изображение](https://github.com/IATkachenko/HA-YandexWeather/assets/17473830/55f2b600-685a-46b4-8630-5a1640932deb)

**Reply by Nikolay-Ch** [reply]
После нескольких перезагрузок ХА - появилась интеграция, спасибо!
